layers/meta-balena-beaglebone/recipes-kernel/linux/linux-bb.org_git.bbappend

```diff
@@ -1 +1,27 @@
 inherit kernel-balena
+
+BRANCH:aarch64 = "v6.1.46-ti-arm64-r7"
+SRCREV:aarch64 = "51c09b7d81b4531f4846cb6c24f46ac239acd69d"
+PV:aarch64 = "6.1.46+git${SRCPV}"
+
+KERNEL_DEVICETREE:aarch64 = " \
```

Comment:
aarch64 might probably be too broad here.

Reply:
Thanks @perezmeyer : Any suggestions on how to narrow this down ? (can I actually use the `beaglebone-play*` targets ?